aait/backend/group-2B/Application.Tests/Features/Authentication/Handlers/Commands/RegisterUserCommandHandlerTest.cs

```diff
@@ -117,13 +121,124 @@ public async Task Invalid_DuplicateEmailRegisterNewUsew()
             CancellationToken.None
         );
 
+        // Change the username (username and email need to be unique)
+        newUser.Username = "new_username";
```

Comment:
This line may be unnecessary. you can create the new user with userName ="new_user"